ssl/ssl_lib.c

```diff
@@ -82,30 +82,36 @@ struct ssl_async_args {
     SSL *s;
     void *buf;
     int num;
-    enum { READFUNC, WRITEFUNC,  OTHERFUNC} type;
+    enum { READFUNC, WRITEFUNC, OTHERFUNC } type;
     union {
-        int (*func_read)(SSL *, void *, int);
-        int (*func_write)(SSL *, const void *, int);
-        int (*func_other)(SSL *);
+        int (*func_read) (SSL *, void *, int);
+        int (*func_write) (SSL *, const void *, int);
+        int (*func_other) (SSL *);
     } f;
 };
 
 static const struct {
     uint8_t mtype;
     uint8_t ord;
-    int     nid;
+    int nid;
 } dane_mds[] = {
-    { DANETLS_MATCHING_FULL, 0, NID_undef },
-    { DANETLS_MATCHING_2256, 1, NID_sha256 },
-    { DANETLS_MATCHING_2512, 2, NID_sha512 },
+    {
+        DANETLS_MATCHING_FULL, 0, NID_undef
+    },
+    {
+        DANETLS_MATCHING_2256, 1, NID_sha256
+    },
+    {
+        DANETLS_MATCHING_2512, 2, NID_sha512
+    },
```

Comment:
I may have figured out why indent seems inconsistent with these kinds of lines. It seems like a line like this will be reformatted as above:

```
{ FOO, 1 } 
```

While the following will not : 

```
{FOO, 1}
```

The difference is the spaces immediately inside the braces.